installing/installing_bare_metal/upi/installing-bare-metal.adoc

```diff
@@ -0,0 +1,259 @@
+
+include::modules/installation-user-infra-machines-advanced-enabling-serial-console.adoc[leveloffset=+3]
+
+include::modules/installation-user-infra-machines-advanced-customizing-iso-or-pxe.adoc[leveloffset=+3]
+
+:boot-media: ISO image
+:boot: iso
+include::modules/installation-user-infra-machines-advanced-customizing-live.adoc[leveloffset=+3]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-serial-console.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-ca-certs.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-network-config.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-iscsi-manual.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-iscsi-ibft.adoc[leveloffset=+4]
+:boot-media!:
+:boot!:
+
+:boot-media: PXE environment
+:boot: pxe
+include::modules/installation-user-infra-machines-advanced-customizing-live.adoc[leveloffset=+3]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-serial-console.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-ca-certs.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-network-config.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-iscsi-manual.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-iscsi-ibft.adoc[leveloffset=+4]
+:boot-media!:
+:boot!:
+
+include::modules/installation-user-infra-machines-static-network.adoc[leveloffset=+3]
+
+include::modules/rhcos-enabling-multipath.adoc[leveloffset=+2]
+
+//iscsi using `coreos-installer install`
+include::modules/rhcos-install-iscsi-manual.adoc[leveloffset=+2]
+
+include::modules/rhcos-install-iscsi-ibft.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../../installing/installing_bare_metal/upi/installing-bare-metal.adoc#creating-machines-bare-metal_installing-bare-metal[Installing {op-system} and starting the {product-title} bootstrap process] for more information on using special `coreos.inst.*` arguments to direct the live installer.
+
+include::modules/installation-installing-bare-metal.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../../support/troubleshooting/troubleshooting-installations.adoc#monitoring-installation-progress_troubleshooting-installations[Monitoring installation progress] for more information about monitoring the installation logs and retrieving diagnostic data if installation issues arise.
+
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+
+include::modules/installation-approve-csrs.adoc[leveloffset=+1]
+
+include::modules/installation-operators-config.adoc[leveloffset=+1]
```

Comment:
If you are really ambitious in a follow-up, I think the guidance (for) now is to [mention that a link goes to an external website](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute). There are one or more links with _Certificate Signing Requests_, should be something like [Certificate Signing Requests] (Kubernetes documentation)